lib/package-manager/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const npm = require('./npm');
+const yarn = require('./yarn');
+
```

Comment:
we should call get executable here for both npm and yarn and cache the results, that way we don't need to recalculate it on subsequent installs.